cleanrl/tqc_continuous_action.py

```diff
@@ -0,0 +1,363 @@
+# docs and experiment results can be found at https://docs.cleanrl.dev/rl-algorithms/sac/#sac_continuous_actionpy
+import argparse
+import os
+import random
+import time
+from distutils.util import strtobool
+
+import gym
+import numpy as np
+import pybullet_envs  # noqa
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.optim as optim
+from stable_baselines3.common.buffers import ReplayBuffer
+from torch.utils.tensorboard import SummaryWriter
+
+PAPER_N_QUANTILES_TO_DROP = {
+    "Hopper-v3": 5,
+    "Swimmer-v3": 2,
+    "HalfCheetah-v3": 0,
+    "Ant-v3": 2,
+    "Walker2d-v3": 2,
+    "Humanoid-v3": 2,
+    "HopperBulletEnv-v0": 5,
+    "SwimmerBulletEnv-v0": 2,
+    "HalfCheetahBulletEnv-v0": 0,
+    "AntBulletEnv-v0": 2,
+    "Walker2dBulletEnv-v0": 2,
+    "HumanoidBulletEnv-v0": 2,
+}
+
+
+def parse_args():
+    # fmt: off
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--exp-name", type=str, default=os.path.basename(__file__).rstrip(".py"),
+        help="the name of this experiment")
+    parser.add_argument("--seed", type=int, default=1,
+        help="seed of the experiment")
+    parser.add_argument("--torch-deterministic", type=lambda x: bool(strtobool(x)), default=True, nargs="?", const=True,
+        help="if toggled, `torch.backends.cudnn.deterministic=False`")
+    parser.add_argument("--cuda", type=lambda x: bool(strtobool(x)), default=True, nargs="?", const=True,
+        help="if toggled, cuda will be enabled by default")
+    parser.add_argument("--track", type=lambda x: bool(strtobool(x)), default=False, nargs="?", const=True,
+        help="if toggled, this experiment will be tracked with Weights and Biases")
+    parser.add_argument("--wandb-project-name", type=str, default="cleanRL",
+        help="the wandb's project name")
+    parser.add_argument("--wandb-entity", type=str, default=None,
+        help="the entity (team) of wandb's project")
+    parser.add_argument("--capture-video",
+        type=lambda x: bool(strtobool(x)), default=False, nargs="?", const=True,
+        help="weather to capture videos of the agent performances (check out `videos` folder)")
```

Comment:
Style nit: please keep this part consistent with other scripts. The first line is the argument, and the second line is the help text / documentation

Reply:
Updated